recipe/build.sh

```diff
@@ -4,11 +4,10 @@ set -exo pipefail
 # Copy tiledb-patches to the source directory
 cp -r "${RECIPE_DIR}/tiledb-patches/." "${SRC_DIR}"
 
-# Use CC/CXX wrappers to disable -Werror
-export NN_CXX_ORIG=$CXX
-export NN_CC_ORIG=$CC
-export CXX=$RECIPE_DIR/cxx_wrap.sh
-export CC=$RECIPE_DIR/cc_wrap.sh
+# Disable -Werror
+export CFLAGS="${CFLAGS//-Werror/}"
```

Comment:
Who sets `-Werror`? Conda?

Reply:
Presumably at least one of the conda-provided compilers set it at some point. The commit that added them (https://github.com/conda-forge/tiledb-feedstock/commit/d98ea04571db4119ea6259effa46de45ce68063f) doesn't say exactly

My guess is that `-Werror` is probably no longer set, but attempting to remove it is a good safeguard, serves as documentation should this ever become an issue again, and is simpler than maintaining the wrapper scripts

Reply:
It was being set in sub-projects (AWS SDK for one, IIRC).

Reply:
> Presumably at least one of the conda-provided compilers set it at some point.

We never did that, that would have been extremely disruptive.

Reply:
> We never did that, that would have been extremely disruptive.

@h-vetinari Agreed. I was mistaken. @ihnorton confirmed that the flag came from the AWS SDK. This was from years ago when we were building the AWS SDK from source. Now that we install the AWS SDK packages from conda-forge, we no longer need to worry about this.